CHAOSScon/2023NA/schedule.md

```diff
@@ -2,4 +2,14 @@
 
 ### May 9, 2023 from 1:00 pm - 5:30 pm Pacific Daylight Time 
 
-Schedule is coming soon! 
\ No newline at end of file
+| Time | Topic | Speaker |
+| --- | -- | --- |
+| 1:00pm | Check-in, networking, and coffee/tea |

```

Comment:
```suggestion
| 1:00-1:30 | Check-in, networking, and coffee/tea |

```

Reply:
This is a total nit, but I think it would be good to be consistent with how we display the times.